src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
+const employeeList = document.querySelector('ul');
+
+function salaryParse(salary) {
+  const cleanSalary = salary.replace(/[^0-9.]/g, '');
```

Comment:
The `salaryParse` function assumes that the salary string will always contain valid numeric characters. Consider adding error handling for cases where the salary string might be empty or invalid.